### ISSUE TITLE
Formula extraction: Multi-line C-style comments are not filtered

### ISSUE THREAD

**Issue by AlexanderSchultheiss** [VariantSync/DiffDetective#97]
# Description
The [CPPDiffLineFormulaExtractor](https://github.com/VariantSync/DiffDetective/blob/f104bdbba88a53f8c87bfcb74535b235410c34e1/src/main/java/org/variantsync/diffdetective/feature/CPPDiffLineFormulaExtractor.java#L21) filters comments in lines containing C preprocessor statements. Currently, only comments are filtered that are fully included in the line. Multi-line comments are not filtered. 

This is a bug because it leads to invalid formulas. 

# Example
The extractor determines the invalid formula `A&&BMulti-lineComment` for the code snippet
```c
#if A && B /* Multi-line
                Comment
                */
```
The expected result is `A&&B`.

# Fix
Multi-line comments need to be filtered after inline comments have been filtered.

**Comment by AlexanderSchultheiss** [comment]
I fixed the bug in 5f6f38d.

**Comment by pmbittner** [comment]
Thanks for reporting the bug and fixing it. The changes look good. Would you like to open a pull-request (targeting the `develop` branch)?
